orbs/jobs/run.yml

```diff
@@ -31,6 +31,10 @@ parameters:
     type: string
     default: latest
     description: "Tool docker image tag"
+  docker_path:
```

Comment:
What do you think on a more descriptive name? Like `docker_archive_path` or `docker_tar_path` or `docker_tar_archive_path`
it seems to be the path of the docker binary

Reply:
I would say `docker_tar_path` what do you think?

Reply:
`docker_tar_path` is fine.